bzt/modules/aggregator.py

```diff
@@ -43,13 +43,27 @@ def __init__(self, low, high, sign_figures):
         self._cached_perc = None
         self._cached_stdev = None
 
+    def __deepcopy__(self, memo):
+        new = RespTimesCounter(self.low, self.high, self.sign_figures)
+        new._cached_perc = self._cached_perc
+        new._cached_stdev = self._cached_stdev
+
+        # TODO: maybe hdrpy can encapsulate this itself
+        new.histogram.counts = copy.deepcopy(self.histogram.counts, memo)
+        new.histogram.total_count = self.histogram.total_count
+        new.histogram.min_value = self.histogram.min_value
+        new.histogram.max_value= self.histogram.max_value
+
+        return new
+
     def __bool__(self):
         return len(self) > 0
 
     def __len__(self):
         return self.histogram.total_count
 
     def add(self, item, count=1):
+        item = round(item * 1000.0, 3)
```

Comment:
This is to keep whole object on a single measurement unit. Before, part of methods were operating with int ms, part with float s.

Reply:
good idea to have it in one place